### ISSUE TITLE
Who should have direct write access / how do we handle pull requests?

### ISSUE THREAD

**Issue by d70-t** [eurec4a/meta#9]
From my point of understanding, there are basically two opposing forces on how to manage write access to this repository. First, the metadata repository should be sourced by the EUREC4A community, so **everyone should be able to contribute** and edit the repository. On the other hand, the data should be used by many people so any changes should go at least through a **small amount of review**. This way, the chance of accidentally breaking other peoples scripts is reduced. This review could be provided through the discussion phase of a pull request.

This gives rise to the following questions:
* [ ] should we use pull requests?
  - [ ] should we enforce the use of pull requests for every edit?
  - [ ] who should moderate / accept  / reject pull requests?
* [ ] who should have write access to the repository?

I would offer moderation for now, but most likely having only one person is not a good idea. Also having too many may lead to a chaotic experience.

What are your opinions on this?

**Comment by bjorn-stevens** [comment]
Yes, I think using the pull request is a good idea, and if it becomes unmanageable we can change tact.  Speaking of mangeable I am still a bit disoriented on GitHub in terms of knowing what sources what, when what gets updated, who does it etc... so this will take practice, and practice takes time, and time is my most in demand resource... so hopefully I find my way soon

**Comment by heikekonow** [comment]
I would also vote for pull requests for the reasons you mentioned, Tobi. Could there be a small team (3-4) that does the moderation? It is always nicer to distribute the work. And between three or four people, the procedures could probably still be easily synchronised.

**Comment by d70-t** [comment]
Yes, I would strongly vote for a moderating team, doing the things alone will most probably be overwhelming and accordingly lead to longer delays. 3-4 people seems to be a reasonable number. Any volunteers?
